srv/components/misc_pkgs/kafka/start.sls

```diff
@@ -17,10 +17,9 @@
 
 
 Start zoopkeper:
-  cmd.run:
-    - name: ./bin/zookeeper-server-start.sh -daemon config/zookeeper.properties
-    - cwd: /opt/kafka
-    - unless: test 1 -le $(ps ax | grep java | grep -i QuorumPeerMain | grep -v grep | awk '{print $1}' | wc -l)
+  service.running:
```

Comment:
Don't delete
Maintain the commands commented, just in case we have to fallback.